tasks/85_bet-validation/bet-validation.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line camelcase
+export function validate_bet(range, bet) {
+  const rangeOfNumbers = range;
+  let result = bet;
+  rangeOfNumbers[0] > rangeOfNumbers[1]
+    ? rangeOfNumbers.reverse()
+    : rangeOfNumbers;
```

Comment:
ternary operator is used to assign value. The result of ternary operator is ignored here.